loaders/httpfs/loader.go

```diff
@@ -0,0 +1,30 @@
+package httpfs
+
+import (
+	"io"
+	"net/http"
+)
+
+type httpFileSystemLoader struct {
+	http.FileSystem
+}
+
+func NewLoader(fs http.FileSystem) httpFileSystemLoader {
```

Comment:
Ya, was thinking about this. i prefer that we move the osFileSystemLoader into the jet package since this is the default loader and will be imported any way.